tools/ontology-builder/src/all_ontology_generator.py

```diff
@@ -266,8 +267,8 @@ def _parse_ontologies(
         version = ontology_info[onto.name]["version"]
         output_file = os.path.join(output_path, get_ontology_file_name(onto.name, version))
         logging.info(f"Processing {output_file}")
-
-        onto_dict = _extract_ontology_term_metadata(onto)
+        allowed_ontologies = [onto.name] + ontology_info[onto.name].get("additional_ontologies", [])
```

Comment:
Should `allowed_ontologies` agree with `additional_ontologies` in the schema?

Reply:
I think allowed_ontologies == main ontology + additional ontologies makes intuitive sense as-is